src/app/store.ts

```diff
@@ -1,10 +1,15 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
+import { todosSlice } from '../features/todos';
+import { filterSlice } from '../features/filter';
+import { currentTodoSlice } from '../features/currentTodo';
+import { TypedUseSelectorHook, useSelector } from 'react-redux';
 
-const rootReducer = combineSlices();
+const rootReducer = combineSlices(todosSlice, filterSlice, currentTodoSlice);
```

Comment:
The function `combineSlices` is incorrect. It should be `combineReducers` from `@reduxjs/toolkit`. Make sure to import `combineReducers` and use it to combine the slices into a root reducer.